Refresh.GameServer/Database/GameDatabaseContext.Relations.cs

```diff
@@ -192,6 +192,81 @@ public void ClearQueue(GameUser user)
 
     #region Rating and Reviewing
 
+    public void RateReview(GameReview review, RatingType rt, GameUser user)
```

Comment:
Variable names shouldn't be shortened to this degree. `ratingType` is just fine.